styles/style.css

```diff
@@ -0,0 +1 @@
+body { font-family: 'Gotham Pro', sans-serif;}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887703)